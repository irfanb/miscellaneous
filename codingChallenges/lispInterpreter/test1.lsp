23728376
:plop
"3230csdjsh  jcjsnkd the quick brown fox jumped over the lazy dog"
hereIsAnIdentifier22
(expression)
(1)
()
