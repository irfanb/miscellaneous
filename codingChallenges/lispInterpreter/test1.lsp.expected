found number 23728376
found SYMBOLATOM :plop
found STRINGATOM "3230csdjsh  jcjsnkd the quick brown fox jumped over the lazy dog"
found id hereIsAnIdentifier22
found EXPRESSION (expression)
found EXPRESSION (1)
found empty EXPRESSION ()
found EXPRESSION (123456)
found EXPRESSION (:sa)
